lib/src/visitor/async_evaluate.dart

```diff
@@ -1706,7 +1707,8 @@ final class _EvaluateVisitor
             return (stylesheet, importer: importer, isDependency: isDependency);
           }
         }
-      } else {
+      }
+      if (_nodeImporter != null) {
```

Comment:
Empty line between `if` statements.

Reply:
Addressed in https://github.com/oddbird/dart-sass/commit/15e33b9b2fc0a9848076f2356a1c99680f8198bc